builder/image/Dockerfile

```diff
@@ -38,9 +38,6 @@ RUN useradd -d $GITHOME $GITUSER
 RUN mkdir -p $GITHOME/.ssh && chown git:git $GITHOME/.ssh
 RUN chown -R $GITUSER:$GITUSER $GITHOME
 
-# let the git user run `sudo /home/git/builder` (not writeable)
-RUN echo "%git    ALL=(ALL:ALL) NOPASSWD:/home/git/builder" >> /etc/sudoers
```

Comment:
I guess you can also remove the `sudo` package now: https://github.com/Blystad/deis/blob/builder_non-root_docker/builder/image/Dockerfile#L6